release_docs/INSTALL

```diff
@@ -51,10 +51,8 @@ CONTENTS
 *****************************************************************************
 
 1. Obtaining HDF5
-        The latest supported public release of HDF5 is available from
-        https://www.hdfgroup.org/downloads/hdf5/.  For Unix and UNIX-like
-        platforms, it is available in tar format compressed with gzip.
-        For Microsoft Windows, it is in ZIP format.
+        The latest supported public releases of HDF5 are available on
+        https://github.com/HDFGroup/hdf5.
```

Comment:
Is this correct? There seems to be a difference between the archives published via GitHub and those on the website. The releases on GitHub are archived git repositories (they include a `.git` directory) and have a top-level directory named `hdfsrc` instead of `hdf5-${version}`.

Contrary to this file (`release_docs/INSTALL`), the root `README` file points to the `hdfgroup.org` website for downloads.